### ISSUE TITLE
xcodeproj target-diff doesn't identify differences in Bundle Resources

### ISSUE THREAD

**Issue by getaaron** [CocoaPods/Xcodeproj#484]
### Reproduction Steps

1. Create an application with a target and add some files to Copy Bundle Resources
2. Duplicate the target
3. Add some additional files to the duplicated target's Copy Bundle Resources
4. Run `xcodeproj target-diff FirstTarget SecondTarget`

### Expected Behavior

I expect to see a list of the additional files added in step 3.

### Actual Behavior

Empty output:

```
--- []
```

### Notes

Running version 1.4.4 of xcodeproj

**Comment by segiddins** [comment]
At the moment, it looks like `target-diff` only diffs source files :/

**Comment by meekapps** [comment]
For anyone else wondering, this appears to still be an issue for the current latest release, 1.5.6.

**Comment by revolter** [comment]
@segiddins, So it doesn't diff the build settings either?

**Comment by segiddins** [comment]
No, it only will diff the build settings per-file at the moment. Though a PR to improve the command would be most welcome!
